website/docs/usage/saving-loading.md

```diff
@@ -313,6 +313,29 @@ installed in the same environment – that's it.
 | `spacy_lookups`                                   | Group of entry points for custom [`Lookups`](/api/lookups), including lemmatizer data. Used by spaCy's [`spacy-lookups-data`](https://github.com/explosion/spacy-lookups-data) package.                                                                  |
 | [`spacy_displacy_colors`](#entry-points-displacy) | Group of entry points of custom label colors for the [displaCy visualizer](/usage/visualizers#ent). The key name doesn't matter, but it should point to a dict of labels and color values. Useful for custom models that predict different entity types. |
 
+### Loading probability tables into existing models
+
+You can load a probability table from [spacy-lookups-data](https://github.com/explosion/spacy-lookups-data) into an existing spaCy model like `en_core_web_sm`.
+
+```python
+### load_table.py
+import spacy
+from spacy.lookups import load_lookups
+nlp = spacy.load("en_core_web_sm")
+lookups = load_lookups("en", ["lexeme_prob"])
+nlp.vocab.lookups.add_table("lexeme_prob", lookups.get_table("lexeme_prob"))
+```
+
+When training a model from scratch you can also specify probability tables in the `config.cfg`.
```

Comment:
This description doesn't correspond to the example, which is for `lexeme_norm` instead of `lexeme_prob`.

Reply:
The probability tables are called `lexeme_prob`, not `lexeme_norm`. (Both exist and would work in these examples, but this example is intended for `lexeme_prob`, right?)

Reply:
Oh sorry, I misunderstood the first message and thought I should rename to `_norm`. I'll change it back to `_prob`.

Reply:
Changed to `lexeme_prob` and made sure that both examples (`.py` and `.cfg`) are consistent this time 😇